_posts/2015-11-18-Stats-2015.markdown

```diff
@@ -0,0 +1,123 @@
+---
+layout: post
+title:  "2015"
+author: orta
+categories: stats community 
+---
+
+> CocoaPods is a dependency manager only in the sense that Apple is a cell phone manufacturer – it’s just one of their many projects.
+
+--  Ash Furrow - [Building Online Communities](http://ashfurrow.com/blog/building-online-communities/)
+
+Last year, around this time, I started an [annual write-up](/Stats-2014/) for CocoaPods. A lot happens in a project with 10 relatively active developers, and a lot of very welcome drive-by contributors. 
+
+<!-- more -->
+
+## Contributions
+
+We started the year with a change in ownership. [@alloy](https://twitter.com/alloy) had been heading up the project for five years, [announced](/The-captain-leaves-the-bridge) he was stepping back due to becoming a father.
```

Comment:
Btw, you’re using real names below, so might look more consistent to use my real name here as well?